templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "sda.fullname" . }}
+  labels:
+    app: {{ template "sda.fullname" . }}
+spec:
+  ports:
```

Comment:
Explicit `type: ClusterIP`?

Reply:
Service type defaults to ClusterIP

Reply:
Yep (and it depends on context, but I often think being explicit is helpful to increase clarity rather than relying on people knowing default behaviour).